base/serialize.jl

```diff
@@ -116,16 +114,16 @@ function serialize(s::SerializationState, t::Tuple)
         writetag(s.io, LONGTUPLE_TAG)
         write(s.io, Int32(l))
     end
-    for i = 1:l
-        serialize(s, t[i])
+    for e in t
```

Comment:
it's 4 characters... this one-letter habit is harmful, variable names that describe what's going on would be a big improvement